src/main/java/org/apache/commons/compress/compressors/gzip/GzipParameters.java

```diff
@@ -539,6 +550,15 @@ public void setFileNameCharset(final Charset charset) {
         this.fileNameCharset = Charsets.toCharset(charset, GzipUtils.GZIP_ENCODING);
     }
 
+    /**
+     * Establishes the presence of the header flag FLG.FHCRC and its headers crc16 value.
+     *
+     * @param hcrc when true, the header crc16 is calculated and inserted in the gzip header on write; on read it means it was present.
+     */
+    public void setHcrc(boolean hcrc) {
```

Comment:
- This is not a good public API name IMO. "Hcrc" -> "HeaderCrc()". 
- New public and protected elements need Javadoc `@since` tags.